### PR TITLE
Remove assigned_client index and metadata

### PR DESCRIPTION
**Pull request by buchi**

### CODE REVIEW

File: opengever/core/upgrades/20200518084137_cleanup_catalog_indexes_and_metadata/upgrade.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        for del_index in del_indexes:
+            del names[del_index]
+
+        # Rebuild the schema
+        schema = {}
+        for i, name in enumerate(names):
+            schema[name] = i
+
+        self.schema = schema
+        self.names = tuple(names)
```
Comment (mbaechtold) [inline opengever/core/upgrades/20200518084137_cleanup_catalog_indexes_and_metadata/upgrade.py:48]: What does this do? It seems not be used anywhere.

File: opengever/core/upgrades/20200518084137_cleanup_catalog_indexes_and_metadata/upgrade.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+class CleanupCatalogIndexesAndMetadata(UpgradeStep):
+    """Cleanup catalog indexes and metadata.
+    """
+
+    def __call__(self):
+        self.remove_metadata()
+
+    def remove_indexes(self):
```
Comment (mbaechtold) [inline opengever/core/upgrades/20200518084137_cleanup_catalog_indexes_and_metadata/upgrade.py:21]: This seems not to be called.

File: opengever/core/upgrades/20200518084137_cleanup_catalog_indexes_and_metadata/upgrade.py (hunk 1 above)
Comment (buchi) [reply opengever/core/upgrades/20200518084137_cleanup_catalog_indexes_and_metadata/upgrade.py]: oops. should be `catalog._catalog` not `self`.

File: opengever/core/upgrades/20200518084137_cleanup_catalog_indexes_and_metadata/upgrade.py (hunk 2 above)
Comment (buchi) [reply opengever/core/upgrades/20200518084137_cleanup_catalog_indexes_and_metadata/upgrade.py:22]: fixed.

**Comment by buchi** [comment]
Indeed. `Task` does not have an `assigned_client` field, so passing it to `Builder` shouldn't have an effect.

**Pull request merged**